Sources/Slacket/Kitura/Linux.swift

```diff
@@ -78,9 +78,12 @@ extension String {
     func stringByAddingPercentEncoding() -> String? {
         #if os(Linux)
             // from https://github.com/apple/swift-corelibs-foundation/blob/d2dc9f3cf91100b752476a72c519a8a629d9df2c/Foundation/NSURL.swift
-            return self.stringByAddingPercentEncodingWithAllowedCharacters(NSCharacterSet.URLHostAllowedCharacterSet())
+            return self.stringByAddingPercentEncodingWithAllowedCharacters(NSCharacterSet.alphanumericCharacterSet())
```

Comment:
This is a temporary solution because I couldn't find a way to add character to `NSCharacterSet` on Linux. AFAIK `_` character shouldn't be escaped so we need to add it to allowed character set like we do for non-Linux OSes. Correct me if I'm wrong